backend/db/migrations/V1_1_0__create_recipe_table.sql

```diff
@@ -0,0 +1,15 @@
+
+CREATE TABLE IF NOT EXISTS recipe
+(
+    id uuid PRIMARY KEY,
+    output_id uuid,
+    foreign key (output_id) references item(id)
```

Comment:
The items table is named ``items`` and not ``item``
```suggestion
    foreign key (output_id) references items(id)
```